### ISSUE TITLE
modal ignore backdrop property in options after the first time

### ISSUE THREAD

**Issue by cjkumaresh** [twbs/bootstrap#19198]
Once the modal is rendered it somehow caches the backdrop property in options which can't be overridden 
pls check the below link https://jsfiddle.net/cjkumaresh/ww7ekv91/5/

**Comment by twbs-lmvtfy** [comment]
Hi @cjkumaresh!

You appear to have posted a live example (https://fiddle.jshell.net/cjkumaresh/ww7ekv91/5/show/light/), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 102, column 1 thru column 7: End tag for  ```` body ```` seen, but there were unclosed elements.
* line 80, column 1 thru column 42: Unclosed element ```` div ````.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by cjkumaresh** [comment]
Have updated the example.Take a look at this http://jsbin.com/verevi/edit?html,output

**Comment by twbs-lmvtfy** [comment]
Hi @cjkumaresh!

You appear to have posted a live example (http://jsbin.com/verevi/edit), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 19, column 13: [E032](https://github.com/twbs/bootlint/wiki/E032): `.modal-dialog` must be a child of `.modal`
* line 20, column 17: [E032](https://github.com/twbs/bootlint/wiki/E032): `.modal-header` must be a child of `.modal-content`
* line 20, column 17: [E032](https://github.com/twbs/bootlint/wiki/E032): `.modal-body` must be a child of `.modal-content`
* line 20, column 17: [E032](https://github.com/twbs/bootlint/wiki/E032): `.modal-footer` must be a child of `.modal-content`
* line 17, column 5: [E046](https://github.com/twbs/bootlint/wiki/E046): `.modal` elements must have a `tabindex` attribute.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by cjkumaresh** [comment]
updated with respect to the comments
http://jsbin.com/verevi/6/edit?html,output

**Comment by cjkumaresh** [comment]
Got the fix from here https://github.com/twbs/bootstrap/pull/5514
Kindly document this.

**Comment by cjkumaresh** [comment]
Closing the issue

**Comment by cvrebert** [comment]
Duplicate of #19371.
